controllers.js

```diff
@@ -0,0 +1,16 @@
+const {getTopicData} = require('./models')
+const app = require("./app");
+const db = require('./db/connection');
+
+function getTopics(req, res){
+    getTopicData().then((result) => {
+        console.log(result)
+        res.status(200).send(result)
+    })
+    .catch((err) => {
```

Comment:
What possible error could happen here, and what is the benefit of console logging it?